R/var_label.R

```diff
@@ -30,6 +30,21 @@
 #' )
 #' var_label(iris)
 #' var_label(iris, unlist = TRUE)
+#'
+#' # Set labels from dictionary, e.g. as read from external file
+#' # One description is missing, one has no match
+#' description = data.frame(
+#'   name = c(
+#'     "Sepal.Length", "Sepal.Width", "Petal.Length", "Petal.Width",
+#'     "Something"),
+#'   label = c(
+#'     "Sepal length", "Sepal width",  "Petal length", "Petal width",
+#'     "something")
+#' )
+#' var_labels = setNames(as.list(description$label), description$name)
+#' iris_labelled = set_variable_labels(iris, .labels= var_labels, .strict=FALSE)
```

Comment:
please respect tidyverse style, i.e. adding a space before and after =

Reply:
Would also prefer the syntaxe

`iris %>% set_variable_labels(.labels = var_labels, .strict = FALSE)`